app/models/katello/concerns/location_extensions.rb

```diff
@@ -8,6 +8,7 @@ module LocationExtensions
         before_create :set_katello_default
         before_save :refute_katello_default_changed
         before_destroy :deletable?
+        attr_accessible :katello_default
```

Comment:
Do we want people to be able to set/update this field in the API? If not, I'd recommend removing this and using the following in our seed file(s):

```ruby
Location.create!(:name => "Default Location") do |location|
  location.katello_default = true
end
```